pkgs/applications/misc/lttoolbox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, lib, pkgs, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "lttoolbox";
+  version = "3.7.1";
+
+  src = fetchFromGitHub {
+    owner = "apertium";
+    repo = "lttoolbox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-3lHXKtwQSrMGQEGOGr27e3kB2qKkTFZcEzeAnIm89Rg=";
+  };
+  name = "lttoolbox";
+  nativeBuildInputs = [
+    pkgs.autoconf
+    pkgs.automake
+    pkgs.pkg-config
+    pkgs.utf8cpp
+    pkgs.libtool
+  ];
+  buildInputs = [
+    pkgs.libxml2
+    pkgs.libxml2.dev
+    pkgs.icu
+  ];
+  buildFlags = [
+    "CPPFLAGS=-I${pkgs.utf8cpp}/include/utf8cpp"
+  ];
+  configurePhase = ''
+    ./autogen.sh --prefix $out
+  '';
+  doCheck = true;
+  checkPhase = ''
+    ${pkgs.python3}/bin/python3 tests/run_tests.py
+  '';
```

Comment:
I couldn't get it to work with the pytest hook, I think because it does some magic setup in [`run_tests.py`](https://github.com/apertium/lttoolbox/blob/v3.7.1/tests/run_tests.py).